src/main/java/com/tradeshift/flagr/Flagr.java

```diff
@@ -82,4 +83,35 @@ public EvaluationResponse evaluate(EvaluationContext context) {
         }
         return evalResponse;
     }
+
+    public Boolean evaluateBoolean(EvaluationContext context) {
+        try {
+            EvaluationResponse evalResponse = this.evaluate(context);
+            String variant = evalResponse.getVariantKey();
+            if(Boolean.parseBoolean(variant) || variant.toLowerCase().trim().equals("enabled")) {
```

Comment:
what about `true` or does it say anywhere that it should be enabled?

Reply:
It's covered by the `parseBoolean`. `Boolean.parseBoolean(variant)` returns `true` if the value is the string true ignoring the case, else returns false.